apps/server/src/modules/mail/rabbitMqClient.ts

```diff
@@ -0,0 +1,13 @@
+import { RabbitMQModule } from '@golevelup/nestjs-rabbitmq';
+
+import { Configuration } from '@hpi-schul-cloud/commons';
```

Comment:
Instead of accessing the Configuration directly, we can extract the needed values in a seperate file `constants.js` and import them from there.